.github/workflows/ci.yml

```diff
@@ -132,12 +132,14 @@ jobs:
       if: github.event_name != 'pull_request' && startsWith(github.ref, 'refs/tags/') && success()
       run: |
         DOCKER_BUILDKIT=1 docker build --no-cache -t ${{ secrets.IMAGE_NAME }}:stable-latest .
+        docker tag ${{ secrets.IMAGE_NAME }}:stable-latest ${{ secrets.IMAGE_NAME }}:${GITHUB_REF/refs\/tags\//}-edge
```

Comment:
That's not how edge is intended to work, this is same as stable with just a edge name added at last 😕